Blog post: A 80/20 approach for sharing user interview insights with your team (#5363)
PostHog/posthog.com: contents/blog/interview-snapshot-guide.md

---
date: 2023-02-23
title: "An 80/20 approach for sharing user interview insights with your team"
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
author:
  - annika-schmid
featuredImage: ../images/blog/posthog-engineering-blog.png
featuredImageType: full
category: Engineering
tags:
  - Guides
---

> You have recently interviewed a user? Congratulations! You have done a big step towards building a user-centric product. Once you’ve gotten into a habit of continuously interviewing users, you will benefit from a steady stream of user insights that help to inform your product strategy and roadmap.
> 
> To turn interviews into insights, a lot of techniques exist, often driven by traditional scientific methods that involve coding your data, sorting cards and generating themes. While these techniques result in well-grounded insights, they are also time consuming. In fast-moving startups, it’s often more practical to have a 80/20 approach for your research, generate quick insights, build an MVP, measure its success and iterate.
> 
> This is why at PostHog, we use interview snapshots to share insights from interviews with the team. They can be created in a very short amount of time and are a great conversation starter. The method was championed by Teresa Torres in the book [Continuous Discovery Habits](https://www.amazon.co.uk/Continuous-Discovery-Habits-Discover-Products/dp/1736633309). It’s a great book if you are looking for a practical approach to user research, whether you are a product engineer, a product manager or a designer. 
> 
> In the spirit of the Pareto principle, we are going to give you our own quick summary of how we do user interview snapshots at PostHog. We’ve created the following template that you can download and fill out, or you can recreate the structure in Google Docs, Figma or any other tool of your choice.

Reviewer: ```suggestion
Regularly talking to users is an important habit for any team, but you also need an efficient way to share what you learn. Traditional scientific methods areeffective but too time consuming, especially for a fast-moving startup. This is why, at PostHog, we use interview snapshots to share insights from interviews with the team. 

Championed by Teresa Torres in the book [Continuous Discovery Habits](https://www.amazon.co.uk/Continuous-Discovery-Habits-Discover-Products/dp/1736633309), interview snapshots are great conversation starters you can create with minimal effort – an 80/20 method that allows us to gather feedback, generate quick insights, build MVPs, measure their success and iterate. 

In this guide, I'll share how we create user interview snapshots at PostHog – including the following template that you can download and fill out, or recreate in Google Docs, Figma or any other tool of your choice.
```

> Here is how to fill out the interview snapshot template:

Reviewer: ```suggestion
## Interview snapshots explained
```

---
date: 2023-02-23
title: "An 80/20 approach for sharing user interview insights with your team"
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
author:
  - annika-schmid
featuredImage: ../images/blog/posthog-engineering-blog.png
featuredImageType: full
category: Engineering
tags:
  - Guides
---

> Regularly talking to users is an important habit for any team, but you also need an efficient way to share what you learn. Traditional scientific methods areeffective but too time consuming, especially for a fast-moving startup. This is why, at PostHog, we use interview snapshots to share insights from interviews with the team. 

Reviewer: ```suggestion
Regularly talking to users is an important habit for any team, but you also need an efficient way to share what you learn. Traditional scientific methods areeffective but too time consuming, especially for a fast-moving startup. This is why we use interview snapshots to share insights from interviews with the team. 
```

> Regularly talking to users is an important habit for any team, but you also need an efficient way to share what you learn. Traditional scientific methods areeffective but too time consuming, especially for a fast-moving startup. This is why, at PostHog, we use interview snapshots to share insights from interviews with the team. 

Reviewer: ```suggestion
Regularly talking to users is an important habit for any team, but it's wasted if you don't develop an efficient way to share what you learn. Traditional scientific methods are effective but too time consuming, especially for a fast-moving startup. Interview snapshots are our solution.
```

> Championed by Teresa Torres in the book [Continuous Discovery Habits](https://www.amazon.co.uk/Continuous-Discovery-Habits-Discover-Products/dp/1736633309), interview snapshots are great conversation starters you can create with minimal effort – an 80/20 method that allows us to gather feedback, generate quick insights, build MVPs, measure their success and iterate. 

Reviewer: ```suggestion
Championed by Teresa Torres in the book [Continuous Discovery Habits](https://www.amazon.co.uk/Continuous-Discovery-Habits-Discover-Products/dp/1736633309), interview snapshots are great conversation starters you can create with minimal effort.
```

In this guide, I'll share how we create user interview snapshots at PostHog – including the following template that you can download and fill out, or recreate in Google Docs, Figma or any other tool of your choice.

// Annika to create this

## Interview snapshots explained

> To create a user interview snapshot, you need a list of insights from a user interview. Ideally you take those notes during the interview, or you can ask one of your colleagues to do so. 

Reviewer: ```suggestion
Snapshots are an easy-to-digest summary of the insights you glean from interviews. Ideally you take those notes during the interview, or you can ask one of your colleagues to do so – sometimes it's easier if one person talks and the other takes notes. 
```

While taking notes, focus on how the user is using your product, and how they feel while using it. What makes them excited? What frustrates them? Those are the insights that later go into your interview snapshot.

> Another approach is to record the conversation, transcribe and summarise it. This approach is more time consuming, so we don’t recommend it. Though you might want to record the conversation in addition to taking notes, if later on you are looking for word-by-word quotes.

Reviewer: ```suggestion
Another approach is to record the conversation, transcribe and summarise it. I don't recommend this approach as it's less efficient, though a recording is useful as a reference for your notes, or for extracting word-by-word quotes later.
```

Don't be afraid to say 'I' rather than we. It sounds more authentic when you do.

> Once you have your insights available, you can open up your interview snapshot template and start filling it out. From experience, we recommend blocking 30 minutes after your interview to review your notes and fill out the user interview snapshot. Here is a step by step guide on how to fill it out:

Reviewer: ```suggestion
Once you have your insights available, you can open up your interview snapshot template and start filling it out. I recommend blocking 30 minutes after your interview to review your notes and fill out the user interview snapshot while it's fresh in your mind. 

Here is a step by step guide on how to fill out your snapshot:
```

### 1. Note high-level information of your user

> To set the scene, add the user’s name at the top of the interview snapshot. For B2B companies, we also recommend adding the job role and company name. It’s also handy to add the date of the interview and a high-level summary what the interview was about. If available, add a picture of the user and the company logo, if the context is B2B. Believe us, this will spur your memory if you revisit the interview in a couple weeks’ time.

Reviewer: ```suggestion
To set the scene, add the user’s name at the top of the interview snapshot. For B2B companies, I also recommend adding the job role and company name for context. It’s also handy to add the date of the interview and a high-level summary what the interview was about. If available, add a picture of the user and the company logo, if the context is B2B. This may seem superfluous, but I find it helps trigger my memory when I revisit interviews a few weeks after the fact.
```

### 2. Add any relevant background information

> What information about your user is relevant to your company and product? Their age? Their responsibilities in their job role? How their morning routine looks like? Write this information down in the “quick facts” column. We also recommend writing down any previous experience the user has with your product.

Reviewer: ```suggestion
What information about your user is relevant to your company and product? Their age? Their responsibilities in their job role? How their morning routine looks like? Write this information down in the “quick facts” column. I also recommend writing down any previous experience the user has with your product here.
```

### 3. List the opportunities you’ve identified

> Did the user express any pain points or desires in relation to your product or problem area? Write these down in the “Opportunities” column. During the interview, make sure you really get to the bottom of the user’s needs, and avoid the [XY problem](https://xyproblem.info/). Make sure to list problems here, and not feature ideas. 

Reviewer: ```suggestion
Did the user express any pain points or desires in relation to your product or problem area? Write these down in the “Opportunities” column. During the interview, make sure you really get to the bottom of the user’s needs, and avoid the [XY problem](https://xyproblem.info/). Make sure to list problems here, and **not feature ideas.** 
```

Is it worth doing a short boxout on the XY problem? Always nice to keep people on the page and it would add some useful context.

### 4. Note any other interesting insights

If the user mentioned anything interesting that doesn’t fit in the “Opportunities” column, you can add it to “Insights”. This can include anything that isn’t a pain point, a need or a desire. Did the user show any uncommon behaviour? Did they share anything about why they chose your product (or not)? If there is anything you want your colleagues to know about without reading the whole interview transcript, this is the place to put it.

### 5. Add 1-2 memorable quotes

> At the top of the template, there is space to add 1-2 memorable quotes from the interview. This is useful for example if the user shared a particular emotion in the interview you want to relay word by word, and you want to create empathy in your team for this particular issue.

Reviewer: ```suggestion
At the top of the template, there is space to add 1-2 memorable quotes from the interview. This is useful, for example, if the user shared a particular emotion in the interview you want to relay word by word, and you want to create empathy in your team for this particular issue.
```

### 6. Share the interview snapshot with your team

> Take a screenshot of your completed interview snapshot and share it on Slack (or any other messenger platform you use)! At PostHog, we have a #posthog-feedback channel everyone in the company has access to. Sharing the snapshot in such a way is bound to create a conversation, follow up questions or ideas. This is exactly what you want.

Reviewer: ```suggestion
Take a screenshot of your completed interview snapshot and share it on Slack – we have a dedicated channel for them. Sharing the snapshot in such a way is bound to create a conversation, follow up questions or ideas. This is exactly what you want.
```

> The [**XY problem**](https://xyproblem.info/) describes when you ask a user for a solution Y to their problem X. If you build Y without understanding X, you are missing important context that would allow you to build a solution that actually solves the problem.

### 4. Note any other interesting insights

If the user mentioned anything interesting that doesn’t fit in the “Opportunities” column, you can add it to “Insights”. This can include anything that isn’t a pain point, a need or a desire. Did the user show any uncommon behaviour? Did they share anything about why they chose your product (or not)? If there is anything you want your colleagues to know about without reading the whole interview transcript, this is the place to put it.

### 5. Add one or two memorable quotes

At the top of the template, there is space to add one or two memorable quotes from the interview. This is useful, for example, if the user shared a particular emotion in the interview you want to relay word by word, and you want to create empathy in your team for this particular issue.

### 6. Share the interview snapshot with your team

> Take a screenshot of your completed interview snapshot and share it with your team – we have a dedicated Slack channel for them. Snapshots are bound to create a conversation, follow up questions or ideas. This is exactly what you want.

Reviewer: Per Charles's feedback, I think this would be opportunity to add some kind of example of this debate in action at PostHog – e.g. a time we changed something immediately based on an interview / snapshot.

---

> date: 2023-03-06

Reviewer: ```suggestion
date: 2023-03-08
```

> title: "Interview Snapshots: Our 80/20 approach to sharing user insights"

Reviewer: ```suggestion
title: "How to turn user interviews into actionable interview snapshots"
```

rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
author:
  - annika-schmid
featuredImage: ../images/blog/posthog-blog-at-desk.png
featuredImageType: full
category: Inside PostHog
tags:
  - Guides
  - Product engineers
---

> Regularly talking to users is an important habit for any product team, but it's wasted without an efficient way to share what you learn. At PostHog, we favor interview snapshots. Championed by Teresa Torres in the book [Continuous Discovery Habits](https://www.amazon.co.uk/Continuous-Discovery-Habits-Discover-Products/dp/1736633309), we like snapshots because:

Reviewer: ```suggestion
Regularly talking to users is an important habit for any product team, but it's wasted without an efficient way to share what you learn. At PostHog, we favor interview snapshots. Championed by Teresa Torres in the book [Continuous Discovery Habits](https://www.amazon.com/Continuous-Discovery-Habits-Discover-Products/dp/1736633309), we like snapshots because:
```

Here's what a completed snapshot looks like:

![interview-snapshot](../images/interview-snapshots/interview-snapshot-filled-out.jpg)

While taking notes, focus on how the user is using your product, and how they feel while using it. What makes them excited? What frustrates them? Those are the insights that later go into your interview snapshot.

Another approach is to record the conversation then transcribe and summarise it later. This might seem easier, but I don't recommend it because it's more time consuming – remember we're optimizing for efficiency here. You can, of course, still record interviews as a useful reference for your notes, or for extracting direct quotes later.

Instead, I recommend blocking 30 minutes after your interview to review your notes and fill out the user interview snapshot while it's fresh in your mind. This way, each interview becomes a simple 60-minute block – 30 minutes for your interview, and 30 minutes for review.

## Creating your interview snapshot

> Once you've reviewed your notes, you're ready to start filling out your snapshot. You can use the template below, or recreate it in Google Docs, Figma etc.

Author: ```suggestion
Once you've reviewed your notes, you're ready to start filling out your snapshot. You can get a copy from the Interview Snapshot template from [Figma](https://www.figma.com/file/iO6kIZ8kXdMudLtu7kSyc5/PostHog-Interview-Snapshot-Template?node-id=0%3A1&t=LDU8ALaROQyfh6JL-1) or [Google Drive](https://drive.google.com/file/d/1Md-yLMtgGTEfjUzSFgUur5I-5H8lSYZp/view).
```

Reviewer 2: Sorry for Google Drive I meant Google Slides (ie. sharing an editable template is what would be useful). If it's just in Figma, that's fine, we can ignore the PDF version!

Reviewer: I'm making a Slides version. Won't take long.

> The [**XY problem**](https://xyproblem.info/) describes when you ask a user for a solution Y to their problem X. If you build Y without understanding X, you are missing important context that would allow you to build a solution that actually solves the problem.

### 4. Note any other interesting insights

If the user mentioned anything interesting that doesn’t fit in the “Opportunities” column, you can add it to “Insights”. This can include anything that isn’t a pain point, a need or a desire. Did the they use the product in an unexpected way? Did they share anything about why they chose your product (or not)? If there is anything you want your colleagues to know about without reading the whole interview transcript, this is the place to put it.

### 5. Add one or two memorable quotes

At the top of the template, there is space to add one or two memorable quotes from the interview. This is useful, for example, if the user shared a particular emotion in the interview you want to relay word for word, and you want to create empathy in your team for this particular issue.

### 6. Share the interview snapshot with your team

> Take a screenshot of your completed interview snapshot and share it with your team – we have a dedicated Slack channel for them. Snapshots are bound to create a conversation, follow up questions or ideas. This is exactly what you want.

Author: ```suggestion
Take a screenshot of your completed interview snapshot and share it with your team – we have a dedicated Slack channel for them. Snapshots are bound to create a conversation, follow up questions or ideas. This is exactly what you want. At PostHog, we had many cases where an Interview Snapshot convinced people in the team why we should invest in a certain feature, or resurfaced older issues that weren't on our mind anymore.
```

Author: @andyvan-ph Would this sentence work as an example? I read through the comments on previous interview snapshots, and the gist is "oh yeah, we should really do this // I remember this was an issue in the past, here is a GitHub ticket" rather then "Here, I have already fixed this small issue"

Reviewer: Yeah, this works.
